ui/.env.goerli

```diff
@@ -6,8 +6,7 @@ NEXT_PUBLIC_BALANCER_NATION_ETH_POOL_ID="0x0bf37157d30dfe6f56757dcadff01aed83b08
 NEXT_PUBLIC_BALANCER_NATION_ETH_POOL_TOKEN="0x161D5108C721C85F1Af4D4f6f790AaBAa37268e7"
```

Comment:
@wkm97 If we change to the file in `contracts/deployments/goerli.json`, can this `ui/.env.goerli` file be deleted?